src/core_modules/capture-core/components/WidgetTwoEventWorkspace/WidgetWrapper/WidgetWrapper.types.ts

```diff
@@ -0,0 +1,12 @@
+import type { WithStyles } from '@material-ui/core/styles';
+import { WidgetTwoEventWorkspaceWrapperTypes } from '../index';
+import type { ProgramStage } from '../../../metaData';
+
+type PlainProps = {
+    widget: any;
+    linkedStage: ProgramStage;
+    stage?: ProgramStage;
+    type?: typeof WidgetTwoEventWorkspaceWrapperTypes[keyof typeof WidgetTwoEventWorkspaceWrapperTypes];
+};
+
+export type Props = PlainProps & WithStyles<typeof import('./WidgetWrapper.container').styles>;
```

Comment:
Delete this line and instead import PlainProps in src/core_modules/capture-core/components/WidgetTwoEventWorkspace/WidgetWrapper/WidgetWrapper.container.tsx and use PlainProps & WithStyles<typeof styles>;